### DISCUSSION TITLE
How to customize the server to return the previous loading page in nuxt3.0

### DISCUSSION THREAD

**Discussion by xuyifeiOnly** [nuxt/nuxt#22490] [Questions]
### Describe the feature

In Nuxt3.0, I want the server's resources to have a loading page before returning, but I don't know how to configure or write it

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by danielroe** [comment]
Could you explain a bit more about what you mean, and what you are trying? Is this in dev mode? Or production?

**Reply by xuyifeiOnly** [reply]
In production mode, I use server-side rendering. After deployment to the production environment, there will be a white screen for the first time when accessing directly. I want to add a loading page, just like in the development environment, where there will be a loading page
![WeChatc530e9b78455847f92239368e256fc59](https://github.com/nuxt/nuxt/assets/49358706/fc6834ca-cd95-4ba7-bb30-6117ec1b7002)

**Reply by danielroe** [reply]
It is supported with a file in `~/app/spa-loading-template.html`.

**Comment by xuyifeiOnly** [comment]
Do you need additional configurations? The following is my loading page, but I did not see the loading page after deploying it to the official environment
<img width="915" alt="image" src="https://github.com/nuxt/nuxt/assets/49358706/dfaf49b0-876b-462a-99a4-58aab90a0df5">
<img width="1217" alt="image" src="https://github.com/nuxt/nuxt/assets/49358706/cc3cac7e-e3f7-4e0f-8fae-5e97e6b59fac">

**Reply by xuyifeiOnly** [reply]
This should be a single page configuration, I am currently rendering the page on the server side
